Gen3_Source_injection.ipynb

```diff
@@ -0,0 +1,602 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def get_lens_radec(cat, butler, region=None, time=False, skymap=None):
</pre><p>can you describe the input, in particular the 'cat' part since it needs to satisfy a specific format</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/DC2-notebooks/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>